src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDeposit.scala

```diff
@@ -167,6 +167,9 @@ object AddDatasetMetadataToDeposit {
         case (Some(q), None,   Some(t)) => elem(s"dcterms:$q")(t)
         case (None,    Some(l),_      ) => elem(s"dc:relation")(l)
         case (None,    None,   Some(t)) => elem(s"dc:relation")(t)
+        case _                          =>
+          // TODO this case needs to be checked to not occur in the preconditions (see also line 32)
```

Comment:
Gave the solution by @jo-pol some thought and I think this is the best solution for now. Instead of the link inside this PR, I used the link to master, which already contains this TODO comment.